src/encoding.md

```diff
@@ -225,6 +226,49 @@ to be percent-encoded. The choice of this set depends on context. For example,
 The return value of encoding is an iterator of `&str` slices which can be
 collected into a `String`.
 
+[ex-base64]: #ex-base64
+<a name="ex-base64"></a>
+## Encode and decode base64
+
+[![base64-badge]][base64] [![cat-encoding-badge]][cat-encoding]
+
+The [`base64`] crate provides `encode` and `decode` functions.
+The example below encode a byte slice to a `base64` encoded `String`
```

Comment:
I would make this whole description more brief. Something like 
"Byte slice is incoded into `base64` String with help of `encode` and subsequently decoded with `decode`."